simplq/src/components/common/Tour/Tour.jsx

```diff
@@ -24,9 +26,9 @@ export default (props) => {
       if (stepNumber === 0) {
         leftArrow.childNodes[0].style.color = 'grey';
         rightArrow.childNodes[0].style.color = 'white';
-      } else if (stepNumber === props.toursteps.length - 1) {
+      } else {
         leftArrow.childNodes[0].style.color = 'white';
-        rightArrow.childNodes[0].style.color = 'grey';
+        rightArrow.childNodes[0].style.color = 'white';
       }
     } else {
       Sentry.captureMessage('left-arrow or right-arrow selectors of reatTour package not found');
```

Comment:
Let us remove this entire function. Looks hacky, and the default behaviour looks good to me.

Reply:
![image](https://user-images.githubusercontent.com/32306490/116810110-68988180-ab5f-11eb-8f88-0eee84692daa.png)

In the default behaviour, the active 'next' icon is more dull than the disabled 'prev' icon. Shouldn't it be the opposite?

Reply:
Yes, it should be the opposite. In that case, this function will have to stay unless there's a better possible implementation.

Reply:
Yeah, I tried locally. 

Right now it's a hacky implementation, and if you go back from the last tutorial, the arrow highlighting is not correct. I would prefer the fix the root cause than fix a bug on top of a hacky implementation.

@maaverik can we remove this function and fix the color as a separate PR?

Reply:
I improved the implementation and fixed the arrow highlighting in the same PR.

Reply:
@mav1n has handled this elegantly in this PR itself. :)